packages/lisk-transaction-pool/src/queue.ts

```diff
@@ -22,18 +32,67 @@ export class Queue {
 		this._index = {};
```

Comment:
should we use this `_` convention for private member?

Reply:
I am defining getter functions for these properties. I am not so happy about it too, but i don't want to have different names for getter functions and variable names, and this seems like the simplest option. Do you have some other suggestion?